nautobot/docs/user-guide/feature-guides/getting-started/ipam.md

```diff
@@ -8,7 +8,7 @@ This next section will demonstrate how to:
 * Assign an IP address to an Interface on a Device
 
 These type of operations fall under an umbrella called IP Address Management (IPAM).
-The Nautobot documentation [IPAM section](../../core-functionality/ipam.md) has more detail on IPAM and each operation.
+The Nautobot documentation [IPAM section](../../core-data-model/ipam/index.md) has more detail on IPAM and each operation.
```

Comment:
This file was deleted since it was just a file full of markdown includes pulling in all of the ipam files that are now under the `core-data-model/ipam` section. Should this line just be removed? 

```
WARNING  -  Documentation file 'user-guide/feature-guides/getting-started/ipam.md' contains a link to 'user-guide/core-data-model/ipam/index.md' which is not found in the documentation files.
```

Reply:
I think so at least for now. I'll fix this in my upcoming PR.

Reply:
I have a desire to have index pages for each of the app sections, but we can do that later.